tests/performance/isValidASCII.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<clickhouse>
+    <query>SELECT isValidASCII(repeat('A', 100)) FROM numbers(1000000) FORMAT Null</query>
```

Comment:
`repeat` function is heavier than `isValidASCII`, so it's better to pre fill tables